apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -0,0 +1,563 @@
+              <button type="submit" class="btn btn-primary">Upload</button>
+            </div>
+          </div>
+        </form>
+      </div>
+    `,
+  });
+}
+
+function AutoAssessmentGroupsModal({
+  groupMin,
+  groupMax,
+  csrfToken,
+}: {
+  groupMin: number;
+  groupMax: number;
+  csrfToken: string;
+}) {
+  return Modal({
+    id: 'autoAssessmentGroupsModal',
+    title: 'Auto new group setting',
+    body: html`
+      <form name="auto-assessment-group-form" method="POST" enctype="multipart/form-data">
+        <div class="form-group">
+          <label for="formMin">Min number of members in a group</label>
+          <input
+            type="text"
+            value="${groupMin}"
+            class="form-control"
+            id="formMin"
+            name="min_group_size"
+          />
+        </div>
+        <div class="form-group">
+          <label for="formMax">Max number of members in a group</label>
+          <input
+            type="text"
+            value="${groupMax}"
+            class="form-control"
+            id="formMax"
+            name="max_group_size"
+          />
+        </div>
+        <div class="d-flex justify-content-end">
+          <div class="form-group mb-0">
+            <input type="hidden" name="__action" value="auto_assessment_groups" />
+            <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+            <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+            <button type="submit" class="btn btn-primary">Group</button>
+          </div>
+        </div>
+      </form>
+    `,
+  });
+}
+
+function AddGroupModal({ csrfToken }: { csrfToken: string }) {
+  return Modal({
+    id: 'addGroupModal',
+    title: 'Add a group',
+    body: html`
+      <form name="add-group-form" method="POST">
+        <div class="modal-body">
+          <div class="form-group">
+            <label for="formName">Group Name</label>
+            <input type="text" class="form-control" id="formName" name="group_name" />
+          </div>
+          <br />
+          <div class="form-group">
+            <label for="formUids">UIDs</label>
+            <input
+              type="text"
+              class="form-control"
+              id="formUids"
+              name="uids"
+              placeholder="one@example.com, two@example.com, three@example.com"
+            />
+            <small id="uidHelp" class="form-text text-muted">
+              Separate with "," <br />
+              Please make sure they are not in any other groups
+            </small>
+          </div>
+        </div>
+      </form>
+    `,
+    footer: html`
+        <input type="hidden" name="__action" value="add_group" />
+        <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+        <button type="submit" class="btn btn-primary">Add</button>
+        </form>
```

Comment:
This shouldn't be here, regardless of if you're using the Modal's form or not.

Reply:
Removed in [dc8a69d](https://github.com/PrairieLearn/PrairieLearn/pull/9550/commits/dc8a69d5bba27345198abea501037c2deb732050).